pkg/security/security.go

```diff
@@ -42,6 +42,15 @@ const (
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
 
+	// GkeWorkloadCertChainFilePath is the well-known path for the GKE workload certificate chain file
+	GkeWorkloadCertChainFilePath = "/var/run/secrets/workload-spiffe-credentials/certificates.pem"
+
+	// GkeWorkloadKeyFilePath is the well-known path for the GKE workload certificate key file
+	GkeWorkloadKeyFilePath = "/var/run/secrets/workload-spiffe-credentials/private_key.pem"
```

Comment:
```suggestion
	GkeWorkloadKeyFilePath = "./var/run/secrets/workload-spiffe-credentials/private_key.pem"
```

Along with the other three. This allows running outside of root and makes things more testable, etc.

In particular, once we do this we can and should have a test for this flow in TestAgent

Reply:
Revised based on the comment and a test has been added for this flow in TestAgent.